.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,86 +1,48 @@
-name: Bug Report
-description: Report a problem with TinyUSB
-labels: 'Bug 🐞'
+name: Bug report
+description: Report build and runtime bugs
+labels: ["Type: Bug"]
 body:
-  - type: markdown
-    attributes:
-      value: |
-        Thanks for taking the time to fill out this bug report!
-        It's okay to leave some blank if it doesn't apply to your problem.
-
-  - type: dropdown
+  - type: checkboxes
+    id: checklist
     attributes:
-      label: Operating System
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
       options:
-              - Linux
-              - MacOS
-              - RaspberryPi OS
-              - Windows 7
-              - Windows 10
-              - Windows 11
-              - Others
-    validations:
-      required: true
-
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
+          required: true
+        - label: I am using target and esp-idf version as defined in component's idf_component.yml
+          required: true
+        - label: I have searched the [issue tracker](https://github.com/espressif/tinyusb/issues?q=is%3Aissue) for a similar issue and not found any related issue.
+          required: true
   - type: input
+    id: idf_version
     attributes:
-      label: Board
-      placeholder: e.g Feather nRF52840 Express
-    validations:
-      required: true
-
-  - type: textarea
-    attributes:
-      label: Firmware
-      placeholder: |
-        e.g examples/device/cdc_msc. If it is custom firmware, it is preferably compiled like one in example folder and reviewable for people to comment on. The easiest way is
-        - Fork this repo, checkout a new branch
-        - Add your-own-example based on stock one
-        - Push and post it here.
+      label: ESP-IDF version.
+      description: On which ESP-IDF version does this issue occur on? Run `git describe --tags` in your esp-idf folder to find it.
+      placeholder: ex. v5.0-rc1
     validations:
       required: true
-
-  - type: textarea
-    attributes:
-      label: What happened ?
-      placeholder: A clear and concise description of what the bug is.
-    validations:
-      required: true
-
-  - type: textarea
+  - type: input
+    id: devkit
     attributes:
-      label: How to reproduce ?
-      placeholder: |
-        Exact steps in chronological order, details should be specific e.g if you use a command/script to test with, please post it as well.
-        1. Go to '...'
-        2. Click on '....'
-        3. See error
+      label: Development Kit.
+      description: On which Development Kit does this issue occur on?
```

I think, here should be either 
```suggestion
      description: Which Development Kit does this issue occur on?
```

or

```suggestion
      description: On which Development Kit does this issue occur?
```